lib/experimental/Forms/Fields/F1SearchBox/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Search } from "@/icons/app"
+import { Input } from "@/ui/input"
+import { ChangeEventHandler, useRef } from "react"
+
+type F1SearchBoxProps = {
+  placeholder?: string
+  value?: string
+  disabled?: boolean
+  threshold?: number
+  debounceTime?: number
+  clearable?: boolean
+  onChange?: (value: string) => void
+}
+
+const F1SearchBox = ({
+  value,
+  threshold = 0,
+  onChange,
+  debounceTime = 0,
```

Comment:
I'm not sure about the debounce. We can use [`useDeferredValue`](https://18.react.dev/reference/react/useDeferredValue) from react, to keep the value "lagging behind" the actual state in case of load. Could it be a good way to do the debounce automagically without people controlling the debounce time?

It could be abused

Reply:
Not sure if this hook do the same I expect, but will check.
I know they can do it by themselves, I just trying to give the feature out of the box to avoid each one implement it in different ways, but we can remove it if you think it could be abused

Reply:
I was checking the useDeferredValue hook and it not the same as a debounce, the hooks defers the value until the new one is ready, by here I'm trying to give developers a functionality to avoid to a lot of network requests